modules/spring-config/src/test/java/org/mule/config/spring/parsers/processors/CheckRequiredAttributesWhenNoChildrenTestCase.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (c) 2003-2015 MuleSoft, Inc. This software is protected under international copyright
```

Comment:
wrong header, this is the EE header but this PR is going to the CE project. Run mvn license:format on the root of your branch to fix